Parking_Lot_Reservation/Models/ParkingSpaceModel.cs

```diff
@@ -10,7 +12,12 @@ public class ParkingSpaceModel
     {
         [Key]
         public int ParkingSpaceId { get; set; }
```

Comment:
No need of adding [Key] attribute. If Name of int variable contains "Id" EntityFramework will come up itself on that.